### ISSUE TITLE
websocket is not working 

### ISSUE THREAD

**Issue by akum42** [Shoonya-Dev/ShoonyaApi-py#60]
I am trying to use Websocket to get latest quotes and my transaction details but working. 
I also tried a sample example given in the repo. Please help

**Comment by TheCodeArtist** [comment]
If you can share the actual steps/code you tried,
and what error messages you get,
then maybe someone can atleast try to understand what problem you are facing

**Comment by akum42** [comment]
any update, i have also added 
`ret = api.start_websocket(order_update_callback=event_handler_order_update, 
            subscribe_callback=event_handler_quote_update, 
            socket_open_callback=open_callback,
            socket_close_callback=close_callback)
            print(ret)

Still getting 
DEBUG:NorenRestApiPy.NorenApi:None
DEBUG:NorenRestApiPy.NorenApi:<websocket._app.WebSocketApp object at 0x7f991a18e438>
websocket closed`

any idea why socket_close_callback is called again and again

**Comment by akum42** [comment]
now I have also added  getting below exception:
```
ret = api.start_websocket(order_update_callback=TradeListenerHelper.event_handler_order_update, 
        subscribe_callback=TradeListenerHelper.event_handler_quote_update,
        socket_open_callback=TradeListenerHelper.open_callback,
        socket_close_callback=TradeListenerHelper.close_callback,
        socket_error_callback=TradeListenerHelper.open_callback)


ERROR:websocket:error from callback <bound method NorenApi.__on_error_callback of <api_helper.ShoonyaApiPy object at 0x7fac2aebccf8>>: open_callback() takes 0 positional arguments but 1 was given
WARNING:NorenRestApiPy.NorenApi:websocket run forever ended in exception, open_callback() takes 0 positional arguments but 1 was given
WARNING:NorenRestApiPy.NorenApi:websocket run forever ended in exception, socket is already opened
WARNING:NorenRestApiPy.NorenApi:websocket run forever ended in exception, socket is already opened
```

**Comment by Shoonya-Dev** [comment]
Greetings from FINVASIA!

Thanks for your Concern.

run this to fix:-  open "/Applications/Python 3.7/Install Certificates.command"
